.github/workflows/pull_requests.yaml

```diff
@@ -55,13 +55,30 @@ jobs:
       - name: compare console values with main
         run: ./dyff --color=off -k between -s <(git show 'origin/main:charts/console/values.yaml') charts/console/values.yaml
   test:
+    name: Run ct tests for ci values matching ${{ matrix.testvaluespattern }} for Redpanda version ${{ matrix.version }}
+    strategy:
+      matrix:
+        version:
+          - ""
+          - v22.3.14
+          - v22.2.11
+        testvaluespattern:
+          - '0[1-3]*'
+          - '0[4-6]*'
+          - '0[7-9]*'
+      fail-fast: true
     runs-on: ubuntu-22.04
     steps:
       - name: Checkout
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
+      - name: Set up for matrix
+        run: |
+          echo bash -O extglob -c "rm -v charts/redpanda/ci/!(${{ matrix.testvaluepattern }})"
+          bash -O extglob -c "rm -v charts/redpanda/ci/!(${{ matrix.testvaluepattern }})"
```

Comment:
Maybe thanks to -x like `bash -O extglob -x -c "rm -v charts/redpanda/ci/!(${{ matrix.testvaluepattern }})"` you could drop echo, right?